source/collections/VectorList.ooc

```diff
@@ -121,9 +121,14 @@ VectorList: class <T> {
 	}
 	getSlice: func (range: Range) -> This<T> {
 		result := This<T> new(range count)
-		result _count = range count
-		source := (this pointer as Long + (range min * (T size))) as Pointer
-		memcpy(result pointer, source, range count * (T size))
+		this getSliceInto(range, result&)
 		result
 	}
+	getSliceInto: func (range: Range, buffer: This<T>@) {
+		if (buffer _vector capacity < range count)
+			buffer _vector resize(range count)
+		buffer _count = range count
+		source := (this pointer as Long + (range min * (T size))) as Pointer
```

Comment:
`LLong: cover from signed long long {` and every signed integer type seems to extend `LLong`, and they are not all the same size. Also, it seems they extend `LLong` to get the functions implemented for `LLong`, but I was honestly not aware covers could extend other covers. I'm very confused now.

Anyway, each type is also a `cover from` a C type, and as far as I know, those types are not guaranteed to have any size. That's why I'm a little worried about the cast to `Long`.